repo/packages/C/chronos/0/marathon.json

```diff
@@ -15,11 +15,8 @@
   "container": {
     "type": "DOCKER",
     "docker": {
-      "image": "mesosphere/chronos:chronos-2.3.2-0.1.20150207000917.ubuntu1404-mesos-0.22.0-1.0.ubuntu1404",
-      "network": "BRIDGE",
-      "portMappings": [
-        { "containerPort": 4400, "hostPort": 0, "servicePort": 4400, "protocol": "tcp" }
```

Comment:
Previously, the host port was bound to a random port obtained from Mesos.  Now, we bind to port 4400 on the host interface.  This is not as good, but permissible as long as you add the following fields to your JSON:

```json
{
  "ports": [4400],
  "requirePorts": true
}
```

Please update this package to bind to a dynamic port ($PORT0), or make the above changes.